src/Viewer/renderers/list.jsx

```diff
@@ -29,10 +29,44 @@ export default React.memo(function List(props) {
   if (SVs.item) {
     return <VisibilitySensor partialVisibility={true} onChange={onChangeVisibility} requireContentsSize={false}><><a name={id} /><li id={id}>{children}</li></></VisibilitySensor>;
   } else if (SVs.numbered) {
-    return <VisibilitySensor partialVisibility={true} onChange={onChangeVisibility}><><ol id={id}><a name={id} />{children}</ol></></VisibilitySensor>;
+    let list_style = styleTypeByLevel[SVs.level]?.numbered;
+    if (!list_style) {
+      list_style = "decimal";
+    }
+    return <VisibilitySensor partialVisibility={true} onChange={onChangeVisibility}><><ol id={id} style={{ listStyleType: list_style }}><a name={id} />{children}</ol></></VisibilitySensor>;
   } else {
-    return <VisibilitySensor partialVisibility={true} onChange={onChangeVisibility}><><ul id={id}><a name={id} />{children}</ul></></VisibilitySensor>;
+    let list_style = styleTypeByLevel[SVs.level]?.unnumbered;
+    if (!list_style) {
+      list_style = "disc";
+    }
+    return <VisibilitySensor partialVisibility={true} onChange={onChangeVisibility}><><ul id={id} style={{ listStyleType: list_style }}><a name={id} />{children}</ul></></VisibilitySensor>;
   }
 
 })
 
+const styleTypeByLevel = {
```

Comment:
Not that we are likely to modify this code, but I probably would have flipped the nesting level here and made the numeric part just a normal array, to avoid the need to retype  the labels "numbered" and "unnumbered" and avoid the need to maintain unique numeric keys manually.

```
styleTypeByLevel = {
   numbered: ["decimal", "upper-roman", "upper-alpha", "lower-roman","lower-alpha", "decimal" ],
   unnumbered: ["square", "circle", "disc", "square", "circle", "disc"]
}